src/middleware.ts

```diff
@@ -149,20 +144,6 @@ export function initializeMiddleware(app) {
 
   // Static assets
   applyStaticAssetMiddlewares(app)
-
-  // Redirects
-  app.get("/collection/:collectionSlug", collectionToArtistSeriesRedirect)
-
-  /**
-   * FIXME: Move this into function
-   * Mount middleware for handling server-side portions of apps mounted into
-   * global router.
-   */
-  app.get("/artist/*", artistMiddleware)
-  app.get("/search*", searchMiddleware)
-
-  // TODO: Artwork download, does this belong here.
-  app.get("/artwork/:artworkID/download/:filename", handleArtworkImageDownload)
```

Comment:
Moved all of this functionality into the new pattern. Code didn't really change at all, just relocated. (Thankful we didn't have much more server-side middleware tho 😅)